### DISCUSSION TITLE
Let's create an Ai protected platform

### DISCUSSION THREAD

**Discussion by longheelslines** [bluesky-social/atproto#2536] [Bluesky Client App]
Hello,

My name is Leo Bailey, I'm an art student at The Animation Workshop, one of the top rated institutions for teaching animation in the world. I have an extensive network here, as well as with students from Gobelins in Paris, the number 1 animation school in Europe.

I'm writing this because of the recent policy changes Meta is planning to integrate on Instagram, which will essentially allow them to train their AI with the content of their creators. 

Obviously, we are a large group of artists, especially young ones, who are very upset about this and the development AI is taking in general. I have been asking around and making posts highlighting these changes, and I have been getting an overwhelming amount of responses - **it seems people are more ready than ever to move to different platforms**. Platforms of tight knit artistic communities, free from greedy, discriminatory algorithms, a platform that blocks AI from scraping content completely!

I know that developers can integrate code that blocks Ai from accessing the site, and there are softwares being developed to poison images, such as Glaze and Nightshade. What we are looking for is a platform that blocks Ai from scraping, prevents Ai content from being posted, and protects user content from being used to train Ai. 

If we can work together to develop a sexy looking social media that can do all these things, I think it could become widely popular, and I believe Bluesky with it's democratic structure could become an integral platform for the creative community. There are already alternatives being built by artists, like the Cara App, but I believe it is possible to make something that looks more attractive!

Let me know what you think, I'm curious to hear suggestions of what can be done!

With all respect,

Leo Bailey

**Comment by MasterJ93** [comment]
Hey! I'm actually thinking of doing some kind of image-related service being built on top of atproto. If you'd like, we can chat and, once I figure out enough pain points, I can factor this into that project. However, if you're willing to help out, you're more than welcome to hop in.

**Reply by longheelslines** [reply]
sounds interesting feel free to update me further

**Comment by syeda7** [comment]
Hello Leo,

Thank you for sharing your thoughts and concerns regarding Meta's policy changes and the broader implications for the artistic community. I completely understand and share your concerns about the potential misuse of artists' content for AI training without explicit consent.

Your proposal for a new platform that blocks AI scraping, prevents AI-generated content, and protects user content is both timely and necessary. Here are some thoughts and suggestions to further this idea:

Technical Implementation:

AI Scraping Prevention: Implementing measures such as robots.txt, CAPTCHA, and obfuscation techniques can help deter automated scraping. Additionally, leveraging tools like Glaze and Nightshade can add an extra layer of protection.
Content Protection: Encrypting images and metadata, along with using digital watermarks, can help ensure that content remains protected.
Community Engagement:

Artist Outreach: Building a community involves reaching out to artists and encouraging them to join and contribute to the platform. Leveraging your network at The Animation Workshop and Gobelins can be a great starting point.
Feedback and Iteration: Regularly gathering feedback from the community and iterating on the platform's features and design based on their input will be crucial for its success.
Platform Features:

User-Friendly Interface: Ensuring the platform is visually appealing and easy to navigate will attract more users. Collaborating with UI/UX designers can help achieve this.
Community Building Tools: Features like forums, discussion boards, and collaboration spaces can help foster a sense of community among artists.
Exploring Existing Platforms:

Bluesky: As you mentioned, Bluesky’s democratic structure could be an excellent foundation. Reaching out to the Bluesky team to discuss potential collaborations might be beneficial.
Cara App: Examining the strengths and weaknesses of existing alternatives like the Cara App can provide insights into what works and what doesn’t.
I believe with a concerted effort and collaboration, we can create a platform that not only protects artists' content but also fosters a vibrant and supportive artistic community. I am keen to hear what others think and look forward to contributing to this initiative.

Best regards,
syeda.

**Comment by forest1102** [comment]
I agree the importance of protecting artists from ai.  

I think invisible watermark for ai protection is one of the solution. But there are people who want to see original images. 
So, like stock photo websites, it could be great idea to implement feature that people can also buy watermark removed images.

**Comment by GhbSmwc** [comment]
Just to let you know, BlueSky is an open, public network by design. If you are planning to have the content you wish to protect on there, I suggest not doing so. Not to mention that decentralized sites functions similarly to peer-to-peer; anything you post will store a copy of it on each nodes that may or may not respect your request to remove/hide it.

Instead, have a preview version (downsized, cropped, watermarked, blurred etc.) on the publicly available as free advertising and then link to whatever protected sites (Patreon, Ko-Fi, Cara, etc.). It's akin to how news sites managed their paywall system (effective if they simply don't load the protected part of the content onto the user's browser). Have part of its content that are publicly available (non-protected) indexable so it can gain SEO.

Yes, I am aware there may be a little tradeoff between allowing easy visibility and AI vs hindering visibility and blocking AI. This tradeoff existed even before this AI problem occurred, [such as robots.txt](https://www.theverge.com/24067997/robots-txt-ai-text-file-web-crawlers-spiders) - 
> The tradeoff is fairly straightforward: if Google can crawl your page, it can index it and show it in search results.

Plus, anything publicly available online (especially not being login/pay-gated) isn't protected, and it shouldn't be, the web is designed to function this way (the web allows any browser, a **user agent**, not a **website agent**, to load an HTML file, and it's the user the one in charge to experience the site, not the web owner), and it's shown [when the people protest against WEI](https://github.com/explainers-by-googlers/Web-Environment-Integrity/issues/28#issuecomment-1642120816).

**Comment by Constantbeginning** [comment]
A simpler integration between Bluesky and Cara could be amazing.

For example: making Bluesky thumbnails (maybe with a watermark)  easier to open an external file in Cara  - since it is a more protected (restrictive) platform.

This way the real file doesn’t leave Cara and is not automatically crawled by the at protocol.

Right now Cara links in Bluesky are huge ugly texts that show a  site preview. 

Working on how to make a better preview, maybe loading a clickable image as link would go a long way -specially if somehow the “real” file can be expanded or opened in a pop up with totally leaving the app.

The question is how to keep this two files associated and how to visualize the protected one outside the at protocol (at least for now).

**Comment by tigeryfan** [comment]
Bluesky [has confirmed](https://bsky.app/profile/bsky.app/post/3lbvgvbviak2c) that they will not be training AI on user data. However, the [Bluesky `robots.txt` file](https://bsky.app/robots.txt) allows all User Agents, and they do not use AI scraping prevention services like [Cloudflare](https://blog.cloudflare.com/declaring-your-aindependence-block-ai-bots-scrapers-and-crawlers-with-a-single-click) to block scrapers. Because Bluesky does not enforce anti-scraping measures, I had to report a HuggingFace dataset with [unconsented user posts recently](https://huggingface.co/datasets/bluesky-community/one-million-bluesky-posts/discussions/13#6746871f72d9e7660955b41b). Thankfully, not many datasets like these exist (and the one I reported was subsequently removed), but stopping a threat early would be the best solution to make Bluesky a refuge for artists fed up with unconsented AI training.

**Reply by tigeryfan** [reply]
It is also worth noting that [the Bluesky ToS](https://bsky.social/about/support/tos) does not contain a clause forbidding the unauthorized and unconsented scraping of posts and user data. I propose that a revision should be done soon.

**Reply by aetos382** [reply]
That statement has nothing to do with having a third party other than the Bluesky administration team use the content posted on Bluesky to train AI.

**Comment by aetos382** [comment]
Here is the place to discuss protocols.
I don't know where is the better place to discuss the policy about the content posted on bsky.app, but at least it should be somewhere like https://github.com/bluesky-social/social-app/issues/2949.

**Comment by vishu42** [comment]
we need this and this seems to be going nowhere
